GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -758,27 +758,46 @@ private void RegisterPlugins()
                 var pluginEntries = PluginRegistry.Plugins
                     .OrderBy(entry => entry.Name, StringComparer.CurrentCultureIgnoreCase);
 
+                // the menu already contains 2 items:
+                //    [0] Plugin Settings
+                //    [1] separator
+                // insert plugins above the separator (the list is in reverse order)
+                int insertOffset = pluginsToolStripMenuItem.DropDownItems.Count - 2;
```

Comment:
This will be incorrect when `RegisterPlugins` is called a second time and the Plugin Manager was added in the previous call.